docker/celery/celeryconfig.py

```diff
@@ -0,0 +1,10 @@
+BROKER_URL = 'redis://redis:6379/1'
+CELERY_RESULT_BACKEND = 'redis://redis:6379/2'
```

Comment:
Maybe, pull up those URIs to environment alongside with reasonable defaults, like
```suggestion
import os

BROKER_URL = os.environ.get('CELERY_BROKER_URL', 'redis://redis:6379/1')
CELERY_RESULT_BACKEND = os.environ.get('CELERY_RESULT_BACKEND', 'redis://redis:6379/2')
```

Reply:
Will be fixed in subsequent commits/PR